analytics/core.go

```diff
@@ -0,0 +1,67 @@
+package analytics
+
+import (
+	"github.com/golang/glog"
+	"log"
+)
+
+type PBSAnalyticsModule interface {
+	LogAuctionObject(*AuctionObject)
+	LogCookieSyncObject(*CookieSyncObject)
+	LogSetUIDObject(*SetUIDObject)
+}
+
+//Collection of all the correctly configured analytics modules - implements the PBSAnalyticsModule interface
+type enabledAnalytics []PBSAnalyticsModule
+
+type factory func(conf map[string]string) (PBSAnalyticsModule, error)
```

Comment:
Not too fond of the generic name "factory" here, but it is private so probably not a big source of confusion. Not asking for a change here, but something to consider for the future. :)